src/chrome/content/rules/CCC.de.xml

```diff
@@ -1,57 +1,44 @@
 <!--
-	For rules subject to CAcert, see CCC.de-cacert.xml.
-
 	Other CCC rulesets:
+		+ C3VOC.de.xml
+		+ CCCv.de.xml
 
-		- C3VOC.de.xml
-		- CCCv.de.xml
-
-
-	Nonfunctional hosts in *ccc.de:
-
-		- dasalte ¹
-		- halfnarp.events ¹
-		- wahlcomputer ²
-
-	¹ Shows wuppa
-
-	Problematic hosts in *ccc.de:
-
-		- wiki.chaosradio²
-		- ds ¹
-		- dsrad ¹
-		- wuppa ¹
-		- irc.darmstadt
```

Comment:
You missed this comment.

Reply:
Which issue does it have?

Reply:
Please include also https://ds.ccc.de/
http://noc.events.ccc.de/ includes the same content as  https://ds.ccc.de/

Reply:
@MilanKral Done.

Reply:
@koops76 just try to access to get the issue: https://irc.darmstadt.ccc.de/

Reply:
@J0WI Done.

Reply:
Please have a look at Travis.